cockroachcloud/architecture.md

```diff
@@ -0,0 +1,76 @@
+---
+title: CockroachCloud Architecture
+summary: Learn more about CockroachCloud architecture
+toc: true
+---
+
+CockroachCloud is a fully-managed deployment of CockroachDB. This page describes CockroachCloud's architecture and how it relates to CockroachDB.
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="serverless">CockroachCloud Serverless </button>
+  <button class="filter-button" data-scope="dedicated">CockroachCloud Dedicated</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+## CockroachCloud Serverless
+
+CockroachCloud Serverless (beta) is a fully-managed, multi-tenant deployment of CockroachDB. Being familiar with the following concepts will help you understand what our Serverless architecture achieves.
+
+### Concepts
+
+Term | Definition
+-----|-----------
+**Serverless cluster** | A cluster that’s automatically billed and scaled in response to the resources it consumes (as opposed to a dedicated cluster, which is billed and scaled statically).
+**Request Unit (RU)** | Request Units represent the compute and I/O resources used by a query. All database operations in CockroachCloud Serverless cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 1 RU, and a "large read" such as a full table scan with indexes could cost 100 RUs. You can see how many Request Units your cluster has used on the [**Cluster Overview**](serverless-cluster-management.html#view-cluster-overview) page.
+**Spend limit** | The maximum amount of money you indicate you want to be billed in a particular billing period for a cluster. The actual amount you are billed is based on the resources used during that billing period. A cluster's budget is allocated across storage and burst performance.
+**Projected usage** | The amount of usage that Cockroach Labs projects a cluster will consume during a billing period. This is important for setting a cluster’s spend limit, because users must leave enough budget to pay for storage for the rest of the billing period.
```

Comment:
Nit (to avoid "users"): "because we always reserve enough budget to pay for storage for the rest of the billing period."